build/Settings.cs

```diff
@@ -341,6 +341,7 @@ public class SignInfo
                 "Microsoft.OData.Edm.dll",
                 "Microsoft.Spatial.dll",
                 "Mono.Posix.NETStandard.dll",
+                "OpenTelemetry.*dll",
```

Comment:
Why do we need to add assembly?

Reply:
the TestPresignedArtifacts test fails since the OpenTelemetry files are not signed (I'm assuming they are added with the new reference to the OOP host)

Reply:
Added each manually